scripts/launch-local-binary.sh

```diff
@@ -101,9 +101,10 @@ sleep 10
 
 # run a litentry-collator instance
 $PARACHAIN_BIN --alice --collator --force-authoring --tmp --chain $CHAIN-dev \
-  --unsafe-ws-external --unsafe-rpc-external --rpc-cors=all \
+  --unsafe-ws-external --unsafe-rpc-external --rpc-cors=all --rpc-methods=Unsafe \
```

Comment:
Does `--unsafe-rpc-external` not suffice?

Reply:
I do find without --rpc-methods=Unsafe, the original setup will throw error in ci-test.

Reply:
By the way, --rpc-methods=Unsafe is not allowed to be used in production environment.
I think it is for different purpose vs  --unsafe-rpc-external
https://github.com/paritytech/polkadot-sdk/issues/68